sympy/liealgebras/SU.py

```diff
@@ -0,0 +1,74 @@
+from sympy.matrices import zeros, ones, Matrix, randMatrix, matrices
+from sympy import mpmath as mp
+from mpmath import qr_solve
+from mpmath import lu_solve
+
+#generator of SU(n) group
```

Comment:
This (or part of it) could belong the docstring. Similarly with other comments below.